src/components/Form.js

```diff
@@ -466,6 +466,7 @@ export default compose(
     withOnyx({
         formState: {
             key: (props) => props.formID,
+            initWithStoredValues: false,
```

Comment:
Could you clarify why this change has been made? The form is used in many places across the codebase, and I'm not sure I understand why this modification has been made.

Reply:
While I was working on this issue I noticed, that keyboard was not opening for Safari when some async calls from Onyx were executed. It was linked to reaching to the browser's local storage - especially when the call was taking ~50 ms. You are 100% right that it doesn't have to be changed for other platforms, so I'll use the `Browser.isMobileSafari()` instead. I'll also add explanation to make clear why it had to be added.